etcdmgmt/etcd-mgmt.go

```diff
@@ -0,0 +1,60 @@
+package etcdmgmt
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+func StartEtcd() error {
+
+	log.Info("Starting etcd")
+	hostname, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Could not able to get hostname")
+		return err
+	}
+
+	HostPort2379 := "http://" + hostname + ":2379"
```

Comment:
We should not start names with caps unnecessarily until and unless they are exposed to other packages. Also I'd like to see a better name here. There is no harm of assigning same value to multiple variables. listenClientUrils, advClientUrls would be better for the sake of better code readability